### PR TITLE
Icon links and buttons need to have a title to show a tooltip | Fixes #972

### LINKED ISSUE #972: Icon links and buttons need to have a title to show a tooltip

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

**Comment by jbalsas** [comment]
@matuzalemsteles, could you please take a look here? 👍

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by matuzalemsteles** [comment]
LGTM!

hey @carloslancha, It seems that the tooltip has some bugs when it is passed the mouse over it does the wrong calculation and the second time, fixes...

http://recordit.co/QKqJaIi0p0

**Comment by carloslancha** [comment]
Hi @matuzalemsteles yep, I already reported it when we merged the component. If you want to take a look would be helpful ;)

Thx!

**Comment by matuzalemsteles** [comment]
Ah yeah, I had not seen it, sorry 😅. I can take a look at this tomorrow 🙂.

File: packages/clay-tooltip/src/ClayTooltip.js [hunk 1]
```diff
@@ -80,17 +80,25 @@ class ClayTooltip extends Component {
...
 			target.removeAttribute('title');
-		} else {
-			title = target.getAttribute('data-title');
+		} else if (target.tagName === 'svg') {
+			let titleTag = target.querySelector('title');
+
+			if (titleTag) {
+				target.setAttribute('data-title', titleTag.innerHTML);
+				target.setAttribute('data-restore-title', 'true');
+				titleTag.remove();
```
Comment (jbalsas) [inline packages/clay-tooltip/src/ClayTooltip.js]: Do we really need to do this for svgs? The only reason we remove the `title` attribute is to make sure it doesn't show the default tooltip. Does it also happens for svgs?

Comment (carloslancha) [reply packages/clay-tooltip/src/ClayTooltip.js]: @jbalsas yep 😅

**Pull request merged**